rsl-learning2022-Risheel/view1/view/app/src/main/res/layout/activity_edit_view.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+    <EditText
+        android:layout_width="300dp"
+        android:layout_height="50dp"
+        android:inputType="date"
+        android:textColor="@color/black"
+        android:background="@color/teal_200"/>
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Date input"
+        android:textSize="30dp"/>
+    <EditText
+        android:layout_width="300dp"
+        android:layout_height="50dp"
+        android:inputType="number"
+        android:textColor="@color/black"
+        android:background="@color/teal_200"/>
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Number input"
+        android:textSize="30dp"/>
+    <EditText
+        android:layout_width="300dp"
+        android:layout_height="50dp"
+        android:inputType="textEmailAddress"
+        android:textColor="@color/black"
+        android:background="@color/teal_200"/>
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Email input"
+        android:textSize="30dp"/>
+    <EditText
+        android:layout_width="300dp"
+        android:layout_height="50dp"
+        android:inputType="text"
+        android:textSize="50dp"
+        android:textColor="@color/black"
+        android:background="@color/teal_200"/>
```

Comment:
EditTexts missing use of `textStyle` and `textSize` in different dimension scales.